addon/components/bootstrap/-checkbox.js

```diff
@@ -3,11 +3,12 @@ import Checkbox from '@ember/component/checkbox';
 export default Checkbox.extend({
   classNames: ['form-check-input'],
 
-  click() {
+  change() {
     const onClick = this.get('onClick');
+    this.set('checked', this.element.checked);
 
     if (onClick) {
-      onClick(!this.get('checked'), this.get('checked'), ...arguments);
+      onClick(!this.element.checked, this.element.checked);
     }
-  }
+  },
```

Comment:
FYI, this is a breaking change, it now requires `change` instead of `onChange` due to a undefined `change` handler being passed to this component from `addon/templates/components/bootstrap-inputs/-checkbox.hbs` if `onChange` isn't defined (which is common).

Reply:
The other option would have been to change the interface of `addon/templates/components/bootstrap-inputs/-checkbox.hbs`.

Reply:
In practice though, `onClick` seems to be used.

Reply:
I can switch this to `onChange` though and update our app, let me know what ya'll think.